server/routes/auth.js

```diff
@@ -1,61 +1,40 @@
 // auth.js
-import passport from 'passport'; 
-import {Strategy, ExtractJwt } from 'passport-jwt';   
-var users = require("./users.js");  
-var cfg = require("./config.js");  
-var params = {  
-    secretOrKey: cfg.jwtSecret,
-    jwtFromRequest: ExtractJwt.fromAuthHeader()
-};
+import models from '../../models/';
```

Comment:
Why is this file in the `routes` folder? Looks to me like it should be with your other `middlewares`